packages/core/src/data-editor/data-editor.tsx

```diff
@@ -651,6 +651,11 @@ export interface DataEditorProps extends Props, Pick<DataGridSearchProps, "image
      * Controls which directions fill is allowed in.
      */
     readonly allowedFillDirections?: FillHandleDirection;
+
+    /**
+     * Determins when a cell is considered activated
```

Comment:
nit: Maybe add a bit more info to this comment about what cell activation actually means. Is it only about the cell overlay being opened?

Reply:
Resolved